bin/module-install

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+try {
+  const opt = require('opt-cli')
+  if (!opt.testOptOut('ghooks-install')) {
+    require('./install')
+  }
+} catch (e) {
+  console.warn( // eslint-disable-line no-console
+    '\nghooks postinstall could not be completed.\n' +
+    '==========================================\n\n' +
+    'Usually this is due to using npm@2.15.x. Please consider updating npm\n' +
+    'to its latest version or run the following command manually afterwards:\n\n' +
+    '\tnode ./bin/install\n'
```

Comment:
👍  As the user will probably run this command from their current working directory.

Do you think it would be useful to include a link to the GH issue?